### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Eternal3l** [anon34342122/nhsm#2] [General]
Just wanted to say thank you for this! I've been wanting to start a new island for a while now without getting rid of my old one and never knew how, Citra's save files for 3DS games have always been much easier to manage so I had no clue what to do for NH on Yuzu. Life saver. Thanks again!!

**Comment by Digerron** [comment]
@mikepound @mhartl @mikeckennedy @mwaskom @excid3 @ly4k @Olical @odrobnik @Earthcomputer @josephwilk @clymb3r @kytrinyx @srowen @owenzhang @owenliang @robb0wen @bigred @buckelij @elijahmanor @Smiles08 @JakeCooper amazing http://vae.fyi
